CONTRIBUTING.md

```diff
@@ -174,40 +157,32 @@ applicable.
 Feel free to contribute standalone pull requests to add new tests or improve
 existing tests as well.
 
-See [Unit testing](https://docs.godotengine.org/en/latest/contributing/development/core_and_modules/unit_testing.html)
-for information on writing tests in Godot's C++ codebase.
+See [Unit testing](https://docs.blazium.app/en/latest/contributing/development/core_and_modules/unit_testing.html)
+for information on writing tests in Blazium's C++ codebase.
 
-## Contributing to Godot translations
+## Contributing to Blazium translations
 
-You can contribute to Godot translations on [Hosted Weblate](https://hosted.weblate.org/projects/godot-engine/),
+You can contribute to Blazium translations on [Hosted Weblate](https://hosted.weblate.org/projects/blazium-engine/),
```

Comment:
Translation contribution is not set up yet

Reply:
Fixed, thank you!